packages/components/inputs/rich-text-utils/src/rich-text-body/rich-text-body.tsx

```diff
@@ -462,18 +572,6 @@ const RichTextEditorBody = forwardRef((props, ref) => {
 });
 
 RichTextEditorBody.displayName = 'RichTextEditorBody';
-RichTextEditorBody.propTypes = {
-  hasError: PropTypes.bool,
-  hasWarning: PropTypes.bool,
-  isReadOnly: PropTypes.bool,
-  isDisabled: PropTypes.bool,
-  editor: PropTypes.any,
-  children: PropTypes.node,
-  containerStyles: PropTypes.any,
-  showExpandIcon: PropTypes.bool.isRequired,
-  onClickExpand: requiredIf(PropTypes.func, (props) => props.showExpandIcon),
```

Comment:
It'll be nice to trigger a warning if props.showExpandIcon is true and onClickExpand is not a function